src/EditorFeatures/VisualBasicTest/Diagnostics/AddImport/AddImportTests.vb

```diff
@@ -121,7 +149,7 @@ NewLines("Class SomeClass \n Sub Foo(Of [|SomeClass|])(x As SomeClass) \n End Su
 NewLines("Class SomeClass \n Sub Foo(Of SomeClass)(x As [|SomeClass|]) \n End Sub \n End Class \n Namespace SomeNamespace \n Class SomeClass \n End Class \n End Namespace"))
         End Function
 
-        <WorkItem(540543, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/540543")>
+        <WorkItem(540543, "http//vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/540543")>
```

Comment:
I didn't...  that's super super weird.  Some other system (hopefully not in roslyn) must have screwed that up!